<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSystemInfoPage</class>
 <widget class="QWizardPage" name="CSystemInfoPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Please Setup your System Information</string>
  </property>
  <property name="subTitle">
   <string>It is recommended that your Prod Directory is local</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QToolButton" name="prodDirBtn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../MainWindow/MainWindow.qrc">
       <normaloff>:/resources/open.png</normaloff>:/resources/open.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Visual Studio Version:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="useCustomCMake">
     <property name="title">
      <string>Do not use CMake Shipped with Visual Studio?</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>CMake Executable:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cmakeExec">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="cmakeExecBtn">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../MainWindow/MainWindow.qrc">
          <normaloff>:/resources/open.png</normaloff>:/resources/open.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="msys64Dir"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Prod Directory:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>MSys 64 Directory:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="prodDir"/>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="vsVersion"/>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="msys64DirBtn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../MainWindow/MainWindow.qrc">
       <normaloff>:/resources/open.png</normaloff>:/resources/open.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>prodDir</tabstop>
  <tabstop>prodDirBtn</tabstop>
  <tabstop>vsVersion</tabstop>
  <tabstop>msys64Dir</tabstop>
  <tabstop>msys64DirBtn</tabstop>
 </tabstops>
 <resources>
  <include location="../MainWindow/MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
